subworkflows/nf-core/bam_variant_calling_somatic_mutect2/main.nf

```diff
@@ -0,0 +1,196 @@
+//
+// Run GATK mutect2 in tumor normal mode, getepileupsummaries, calculatecontamination, learnreadorientationmodel and filtermutectcalls
+//
+
+include { GATK4_CALCULATECONTAMINATION                                      } from '../../../modules/nf-core/gatk4/calculatecontamination/main'
+include { GATK4_FILTERMUTECTCALLS                                           } from '../../../modules/nf-core/gatk4/filtermutectcalls/main'
+include { GATK4_GATHERPILEUPSUMMARIES as GATK4_GATHERPILEUPSUMMARIES_NORMAL } from '../../../modules/nf-core/gatk4/gatherpileupsummaries/main'
+include { GATK4_GATHERPILEUPSUMMARIES as GATK4_GATHERPILEUPSUMMARIES_TUMOR  } from '../../../modules/nf-core/gatk4/gatherpileupsummaries/main'
+include { GATK4_GETPILEUPSUMMARIES    as GATK4_GETPILEUPSUMMARIES_NORMAL    } from '../../../modules/nf-core/gatk4/getpileupsummaries/main'
+include { GATK4_GETPILEUPSUMMARIES    as GATK4_GETPILEUPSUMMARIES_TUMOR     } from '../../../modules/nf-core/gatk4/getpileupsummaries/main'
+include { GATK4_LEARNREADORIENTATIONMODEL                                   } from '../../../modules/nf-core/gatk4/learnreadorientationmodel/main'
+include { GATK4_MERGEMUTECTSTATS                                            } from '../../../modules/nf-core/gatk4/mergemutectstats/main'
+include { GATK4_MERGEVCFS                                                   } from '../../../modules/nf-core/gatk4/mergevcfs/main'
+include { GATK4_MUTECT2                                                     } from '../../../modules/nf-core/gatk4/mutect2/main'
+
+workflow BAM_VARIANT_CALLING_SOMATIC_MUTECT2 {
+    take:
+    input                     // channel: [ meta, [ input ], [ input_index ] ]
+    fasta                     // channel: /path/to/reference/fasta
+    fai                       // channel: /path/to/reference/fasta/index
+    dict                      // channel: /path/to/reference/fasta/dictionary
+    germline_resource         // channel: /path/to/germline/resource
+    germline_resource_tbi     // channel: /path/to/germline/index
+    panel_of_normals          // channel: /path/to/panel/of/normals
+    panel_of_normals_tbi      // channel: /path/to/panel/of/normals/index
+    intervals                 // channel: [mandatory] [ intervals, num_intervals ] or [ [], 0 ] if no intervals
+    joint_mutect2             // boolean: [mandatory] [default: false] run mutect2 in joint mode
+
+    main:
+    versions = Channel.empty()
+
+    germline_resource_pileup     = germline_resource_tbi ? germline_resource : Channel.empty()
+    germline_resource_pileup_tbi = germline_resource_tbi ?: Channel.empty()
+
+    if (joint_mutect2) {
+        // Separate normal cram files and remove duplicates
+        ch_normal_cram = input.map{ meta, input, input_index -> [ meta - meta.subMap('tumor_id') + [id:meta.patient], input[0], input_index[0] ] }.unique()
```

Comment:
@maxulysse : So if we decided to change `meta.patient` to `meta.entity` over in Sarek, then this subworkflow in nf-core/modules will also have to be updated?

Reply:
yeah, or patch it